app/questions/tests/test_questions_routes.py

```diff
@@ -7,6 +7,75 @@
 @pytest.mark.integration
 def test_get_questions(client):
     response = client.get(url_for("questions.get_questions"))
+
+    assert response.status_code == 200
+    assert response.content_type == "application/json"
+    assert response.access_control_allow_origin == "*"
+
+    json_decoded = response.get_json()
+    assert type(json_decoded) == dict
+    assert "SetOne" in json_decoded
+    assert "SetTwo" in json_decoded
+    assert "Answers" in json_decoded
+    assert "Directions" in json_decoded
+
+
+@pytest.mark.integration
+def test_set_one_properties(client):
+    response = client.get(url_for("questions.get_questions"))
+    set_one = response.get_json()["SetOne"]
+
+    assert isinstance(set_one, list)
+    assert len(set_one) == 10
+    for question_obj in set_one:
+        assert type(question_obj) == dict
+        assert "id" in question_obj
+        assert "value" in question_obj
+        assert "question" in question_obj
+        assert type(question_obj["id"]) == int or type(question_obj["id"]) == float
```

Comment:
This should always be an `int`, so it would be better to remove the `float` comparison so that the test is stricter.

Same for the `"id`" assertions on like 52 and 68.